AKS: support for ingress_application_gateway addon (#11376)
terraform-providers/terraform-provider-azurerm: website/docs/d/kubernetes_cluster.html.markdown

* `enabled` - Is Azure Policy for Kubernetes enabled?

An `ingress_application_gateway` block supports the following:

* `enabled` -  Has this cluster been integrated with an application gateway.

* `effective_application_gateway_id` - The ID of the application gateway associated with the application gateway ingress controller deployed to this cluster.

> 

Reviewer: Persumably missing the other attributes here?

* `enabled` - Is Azure Policy for Kubernetes enabled?

An `ingress_application_gateway` block supports the following:

> * `enabled` -  Has this cluster been integrated with an application gateway.

Reviewer: Can we modify the wording a bit here to be consistent with the one in the resource document?
```suggestion
* `enabled` -  Is the Application Gateway ingress controller integrated with this Kubernetes Cluster?
```

> * `effective_gateway_id` - The ID of the application gateway associated with the application gateway ingress controller deployed to this cluster.

Reviewer: Can we modify the wording a bit here to be consistent with the one in the resource document?
```suggestion
* `effective_gateway_id` - The ID of the Application Gateway associated with the ingress controller deployed to this Kubernetes Cluster.
```
